database/prefixdb/db_test.go

```diff
@@ -21,3 +21,13 @@ func TestInterface(t *testing.T) {
 		test(t, NewNested([]byte("ld"), New([]byte("wor"), db)))
 	}
 }
+
+func BenchmarkInterface(b *testing.B) {
```

Comment:
This only benchmarks the `memdb` not the `prefixeddb`. So I think this db should be wrapped. However I think it would also be nice to include a benchmark in the `memdb` package as well,

Reply:
Added `memdb` benchmarks and fixed the `prefixdb` ones to use `New` and `NewNested`.